probablility.py

```diff
@@ -42,12 +42,12 @@ def play(playersQueue, numbersInsideBoxes):
                 # print('Player ' + str(playerNumber) + ' found their number after opening ' + str(numberOfBoxesOpened) + ' boxes!')
                 break
             elif numberOfBoxesOpened == 50:
-                print('Player ' + str(playerNumber) + ' did not find their number while opening ' + str(numberOfBoxesOpened) + ' boxes. All players shall die!')
+                #print('Player ' + str(playerNumber) + ' did not find their number while opening ' + str(numberOfBoxesOpened) + ' boxes. All players shall die!')
```

Comment:
You know what would be megacool 😎
Check if VERBOSE env var is enabled and only then print verbose logs.
This way you don’t have to commit commented out code to git 😌

```
if (os.environ['VERBOSE']):
    print(…
```

then you can just run the script with verbose mode to see the verbose logs
`VERBOSE=true python3 probability.py 10000`

https://able.bio/rhett/how-to-set-and-get-environment-variables-in-python--274rgt5